src/Home/View.fs

```diff
@@ -226,7 +226,7 @@ let root model dispatch =
     section [Class "hero is-small"] [
         div [Class "hero-body"] [
           div[ Class "standard-margin"] [
-            h1 [Class "title is-2 title-bold"] [str "SPONSORS."]
+            h1 [Class "title is-2 title-bold"] [str "SPONSORS"]
```

Comment:
I think the period after the titles was intended by @whitetigle for stylistic reasons as it appears in all the other places in the web. Could you please add it back (also for the pre-conference event above)?